keyboards/keyap/keymaps/via/rules.mk

```
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

Probably want to enable this here.
```suggestion
VIA_ENABLE = yes
ENCODER_MAP_ENABLE = yes
```